vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -9,13 +9,10 @@
           - vbmc
       ironic_containers:
           - ironic
-          - ironic-api
-          - ironic-conductor
           - ironic-inspector
           - ironic-endpoint-keepalived
           - ironic-log-watch
           - dnsmasq
-          - mariadb
```

Comment:
@dtantsur are you suggesting leaving MariaDB in this list? I am not sure why the fetching logs task is complaining about this container if we are still using it.

Reply:
I double checked it, here is flow:

- when KIND used, we do run [RUN_LOCAL_SCRIPT](https://github.com/metal3-io/metal3-dev-env/blob/c539be459382279fb800d4fcf16e0d34c674d5cf/03_launch_mgmt_cluster.sh#L190) in 03_*
-  in run local ironic script we [set](https://github.com/metal3-io/baremetal-operator/blob/main/tools/run_local_ironic.sh#L93) using mariadb container by default to false and will [run](https://github.com/metal3-io/baremetal-operator/blob/main/tools/run_local_ironic.sh#L259-L266) it only when true. 

So, I do not think we need to keep MariaDB container in that list

Reply:
CI also confirms that  `IRONIC_USE_MARIADB=false`, i.e this run: https://jenkins.nordix.org/view/Metal3%20Main/job/metal3_main_v1a5_integration_test_ubuntu/40/consoleFull